tests/infra/crypto.py

```diff
@@ -147,3 +151,40 @@ def verify_request_sig(raw_cert, sig, req, request_body, md):
             raise e
 
         verify_recover_secp256k1_bc(sig, req)
+
+
+def generate_aes_key(key_size: int) -> bytes:
+    return secrets.token_bytes(key_size // 8)
```

Comment:
```suggestion
def generate_aes_key(key_bits: int) -> bytes:
    return secrets.token_bytes(key_bits // 8)
```

Seems a little more readable.